src/o365/outlook/commands.ts

```diff
@@ -2,6 +2,7 @@ const prefix: string = 'outlook';
 
 export default {
   OUTLOOK_MAIL_SEND: `${prefix} mail send`,
+  OUTLOOK_REPORT_MAILAPPSUSAGEAPPSUSERCOUNTS: `${prefix} report mailappusageappsusercounts`,
```

Comment:
Renamed to `OUTLOOK_REPORT_MAILAPPUSAGEAPPSUSERCOUNTS` to match the command name.